Source/WebInspectorUI/UserInterface/Views/Variables.css

```diff
@@ -424,14 +424,10 @@ body {
         }
     }
 
-    &:is(.mac-platform.monterey, .mac-platform.big-sur):not(.docked) {
+    &:not(.docked) {
```

Comment:
shouldn't this be `&.mac-platform:not(.docked)`?

Reply:
This does look contradictory to the specific style for mac platforms defined in Main.js:

https://github.com/WebKit/WebKit/blob/1907a445147bf44d8a54bd25067bc62ecc06b63f/Source/WebInspectorUI/UserInterface/Base/Main.js#L2650-L2655

But I'm hesitant to make the rule more specific because it collapses `#undocked-title-area` to zero on non-Mac platforms and that causes overlap within the UI. Tested by removing `mac-platform` from the class list, I don't know for sure how Web Inspector looks like on a non-Mac platform. <img width="1075" alt="Screenshot 2024-10-27 at 14 36 50" src="https://github.com/user-attachments/assets/a105e03f-330d-4c70-aadc-cf4a2cea7e75">